File: android/app/src/main/java/com/djs66256/short_drama/domain/model/TheaterDrama.kt
package com.djs66256.short_drama.domain.model

data class TheaterDrama(
    val id: String,
    val title: String,
    val description: String,
    val coverUrl: String,
    val category: String,
    val episodeCount: Int,
    val tags: List<String>,
    val rating: Double,
    val createdAt: String,
    val updatedAt: String,
    val heat: Int,
)

File: ios/ShortDrama/Sources/Domain/Entities/TheaterDrama.swift
import Foundation

/// Domain entity representing a theater feed card.
struct TheaterDrama: Identifiable, Equatable, Sendable {
    let id: String
    let title: String
    let description: String
    let coverUrl: String?
    let category: String
    let episodeCount: Int
    let tags: [String]?
    let rating: Double?
    let createdAt: String
    let updatedAt: String
    let heat: Int
}
